test/integration/docker/in_memory_build_context.py

```diff
@@ -0,0 +1,45 @@
+import io
+import tarfile
+import time
+
+
+class InMemoryBuildContext:
```

Comment:
Please confirm: Currently this class is not used, yet.

Reply:
This will be used in #142